src/mods/vr/runtimes/OpenVR.cpp

```diff
@@ -136,11 +136,25 @@ VRRuntime::Error OpenVR::update_render_target_size() {
 }
 
 uint32_t OpenVR::get_width() const {
-    return this->w;
+    auto width = this->w;
+    // if we've altered the default projection matrix we'll be cropping the image - if the image bounds are non-standard and
+    // the setting's enabled, scale the recommended width so the cropped width is the same as the recommended width
+    if (!(m_view_bounds[0][0] == 0 && m_view_bounds[0][1] == 1 && m_view_bounds[1][0] == 0 && m_view_bounds[1][1] == 1) &&
+        VR::get()->should_grow_rectangle_for_projection_cropping()) {
+        width = width / std::max(m_view_bounds[0][1] - m_view_bounds[0][0], m_view_bounds[1][1] - m_view_bounds[1][0]);
+    }
```

Comment:
Not a fan of this code running every time this is called, cache it somehow

Reply:
Good point

Reply:
modified so the eye project matrices, view bounds and texture scaling is only done when it changes